apis/nucleus/src/components/ActionsToolbar.jsx

```diff
@@ -219,7 +215,7 @@ function ActionsToolbar({
           alignTo={moreRef}
           popoverProps={more.popoverProps}
           popoverPaperStyle={more.popoverPaperStyle}
-          onCloseOrActionClick={handleCloseShowMoreItems}
+          onCloseOrActionClick={() => setShowMoreItems(false)}
```

Comment:
The previous is more optimised in general, at least if wrapping the callback in a `useCallback()` func:
```
const handleCloseShowMoreItems = useCallback(() => {
  setShowMoreItems(false);
}, [setShowMoreItems]);
```